cypress/page/item-page.ts

```diff
@@ -0,0 +1,12 @@
+class ItemPage {
+  private itemName: string;
+
+  constructor() {
+    this.itemName = "#add-to-cart-sauce-labs-bolt-t-shirt"
+  }
+  public selectItem(): void {
+    cy.wait(500)
```

Comment:
It is not recommended to add this kind of waits

Reply:
It was deleted. I forgot that for a test